traffic_ops/install/bin/_postinstall

```diff
@@ -1004,6 +1018,14 @@ def generate_cdn_conf(questions, fname, automatic, root): # type: (list[Question
 		json.dump(existing_conf, conf_file, indent=indent)
 		print(file=conf_file)
 	logging.info("CDN configuration has been saved")
+	try:
```

Comment:
That was my thinking behind leaving it as a warning.

Reply:
I have set it back to a warning

Reply:
> but since `traffic_vault_backend` is not a required field, should we actually be raising an exception in this case?

Good point, warning makes sense.